src/Tasks/ResolveManifestFiles.cs

```diff
@@ -665,6 +669,8 @@ private ITaskItem GetOutputEntryPoint(ITaskItem entryPoint, List<PublishInfo> ma
 
         private bool IsFiltered(ITaskItem item)
         {
+            bool isDotNetCore = String.Equals(TargetFrameworkIdentifier, Constants.DotNetCoreAppIdentifier, StringComparison.InvariantCultureIgnoreCase);
```

Comment:
Does this work for .NET 5+?

Reply:
Yes, because it's checking the `TargetFramework`**`Identifier`**, which is still `.NETCoreApp`. That's different from the short name that is usually used in project files:

|short name|`TargetFrameworkIdentifier`|`TargetFrameworkMoniker`|
|--|--|--|
| `netcoreapp3.1`|`.NETCoreApp`|`.NETCoreApp,Version=v3.1`|
| `net5.0`|`.NETCoreApp`|`.NETCoreApp,Version=v5.0`|

Reply:
It does.  We kept TargetFrameworkIdentifier having the .netcoreapp name for compat and consistency reasons even as we changed the primary TargetFramework.  See https://github.com/dotnet/sdk/blob/955c0fc7b06e2fa34bacd076ed39f61e4fb61716/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.TargetFrameworkInference.targets#L103 as an example where we check that value in the sdk.

Reply:
What Rainier said below. .NET 5+ will use .NETCoreApp as its framework identifier.